preprocessing/science/rconv/README.md

```diff
@@ -2,7 +2,30 @@
 Rconv is a small tool which transforms file given in the Standard Rupture Format (SRF) to the intermediate NetCDF Rupture Format (NRF) which is required by SeisSol for simulating kinematic rupture models.
 
 ## Building rconv
-You need to have the proj.4 and the NetCDF libraries installed.
+
+rconv depends on the PROJ and NetCDF libraries.
+
+### Installing PROJ
```

Comment:
Why don't we just state that rconv depends on version 4? Then a user can use an existing installation.